modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/panels/InfoPanelProvider.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, {useCallback, useEffect, useState} from 'react';
+
+function InfoPanelProvider(props) {
+	const {children, onClose, onOpen} = props;
+	const [active, setActive] = useState(false);
+
+	const setInfoPanelActive = useCallback(
```

Comment:
please check other panels active vs open semantic, I find this naming a little confusing. This may be only a concern that I have alone.

Reply:
We have a lot of cases - active, isOpen, open, show,